cli/lib/utils.ts

```diff
@@ -35,23 +41,71 @@ export function paginateQuery<T>(
   return fn
 }
 
+/**
+ * Paginate through an Auth0 record set where the response if of the form Promise<ApiResponse<Record<K, T[]>>>,
+ * with the data nested under response.data[key]
+ */
+export function paginateNestedQuery<K extends string, T>(
```

Comment:
Auth0 has switched to this format for API responses for newly added endpoints (e.g. for actions and trigger bindings), but old endpoints still use the previous format which returns an array directly